mappings/net/minecraft/block/ChestBlock.mapping

```diff
@@ -5,7 +5,10 @@ CLASS bna net/minecraft/block/ChestBlock
 				ARG 1 syncId
 	CLASS bna$a PropertyRetriever
 		METHOD b getFromSingleChest (Lbtq;)Ljava/lang/Object;
+			ARG 1 chest
 		METHOD b getFromDoubleChest (Lbtq;Lbtq;)Ljava/lang/Object;
+			ARG 1 chest1
+			ARG 2 chest2
```

Comment:
maybe leftChest and rightChest?

Reply:
Sure, but I think the order is reversed with rightChest being the 1st param unless I'm misreading lines 176-178

Reply:
Yeah, this was not meant to be the correct order, but a general suggestion, since I didn't look at the code 😉